core/store.go

```diff
@@ -91,9 +91,11 @@ func Get(k string) *Obj {
 	v := store[ptr]
 	if v != nil {
 		if hasExpired(v) {
+			keypoolMutex.RUnlock()
```

Comment:
Just wanted to highlight something... I'm not sure if this worked for me. What worked for me was unlocking the store mutex prior to keypool...because in `Del` - `store` lock is acquired first.

Reply:
Hi @kolharsam, Would it be possible to share what is the outcome if the current implementation is used vs when we release the store mutex first?

Reply:
Usually the order in which the locks are released doesn't matter much.